README.md

```diff
@@ -347,4 +348,28 @@ spec:
       storage: 2Gi
   volumeMode: Filesystem
 EOF
-```
\ No newline at end of file
+```
+
+### Building and publishing the image to quay with buildah
```

Comment:
Can we use the quarkus container image extensions for this?

For example we could use `quarkus-container-image-jib` and require no external tools for the job.

An `mvn clean package -Dquarkus-container-image-push=true` would suffice.

Reply:
I tried it but was unable to. When I launch it I got a docker error, but I'm not sure now how this test was made, so I'll try it again.

Reply:
This fails because it seems that jib is using docker anyway and I don't have a docker installation available.

I launched this...

```bash
$ ./mvnw clean package -Dquarkus-container-image-push=true -Dquarkus.container-image.build=true -Pjib
```
...and this is the result...

```
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:1.6.0.Final:build (default) on project snowdrop-bot: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
...
[ERROR] Caused by: java.util.concurrent.ExecutionException: java.io.IOException: 'docker load' command failed with error: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
...
[ERROR] Caused by: java.io.IOException: 'docker load' command failed with error: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
...
```